src/smart-components/workflow/remove-workflow-modal.js

```diff
@@ -92,7 +105,14 @@ const RemoveWorkflowModal = ({
                     : (<React.Fragment>
                       { ids.length } { intl.formatMessage(worfklowMessages.approvalProcesses) }
                     </React.Fragment>)
-                }</b>
+                }</b>,
+                dependenciesMessageValue:
+                      isEmpty(dependenciesMessage()) ? '.' :
+                        <React.Fragment>
+                            &nbsp;{ intl.formatMessage(worfklowMessages.fromProcessDependencies) }: <br/><br/>
```

Comment:
We should make this just one message. The list can be passed as a variable and the new lines as well.